### ISSUE TITLE
Right-click crashes application

### ISSUE THREAD

**Issue by roberterdin** [maketechnology/chromium.swt#12]
While evaluating the project to make a potential donation I noticed that a right-click into the browser crashes the application in OSX. I assume this is the relevant output in the stdout/stderr:

```
J:get_context_menu_handler
2017-08-09 13:40:40.990 java[28644:521060] -[SWTApplication isHandlingSendEvent]: unrecognized selector sent to instance 0x7fa5865586b0
```
Here is the detailed log:
[hs_err_pid28644.txt](https://github.com/maketechnology/cefswt/files/1211411/hs_err_pid28644.txt)

**Comment by gzunino** [comment]
Hi @roberterdin, thanks for the issue report. We are aware of the right click crash on MacOS and many others. This is due missing implementation of the CEF APIs for handling right click (and potentially delegate to standard SWT menu handling). 
A donation pledge through Eclipse Foundation (Mikael Barbero) will help get us closer to the goal and implement that missing API and many others required for production usage.
At this stage we depend on donations pledges to move the project forward.
Please take a look to README for contacting us and Mikael to offer such a donation !

**Comment by gzunino** [comment]
Should be fixed on 0.5.0
